emsdk.py

```diff
@@ -2842,6 +2832,7 @@ def extract_bool_arg(name):
   arg_old = extract_bool_arg('--old')
   arg_uses = extract_bool_arg('--uses')
   arg_global = extract_bool_arg('--global')
+  arg_system = extract_bool_arg('--system')
```

Comment:
If `arg_system` is provided bu `arg_global` is not, it should automatically set that.
```suggestion
  arg_system = extract_bool_arg('--system')
  if arg_system
  	arg_global=True
```